src/stylesheet-loader.ts

```diff
@@ -4,12 +4,17 @@ import BrowserSupport from './browser-support';
 
 export interface LoadStylesheetOptions {
     prepend: boolean;
+    attributes: StylesheetAttributes;
```

Comment:
should this be marked as optional?

Reply:
I believe this are mandatory, prepend won't serve of any purpose if we don't provide actual attributes.

Reply:
I think prepend is not related to the attributes itself, but rather, where it's loaded?

https://github.com/bigcommerce/script-loader-js/pull/11/files#diff-fad3242aa36c1c27d88b5b7c6a77e7f1R49

Reply:
I checked Script-loader#7 but I think it's probably a typescript bug. Should be marked as optional there too, as there is code that caters for the scenario when `attributes` is missing.